projects/demo/src/modules/experimental/card/examples/7/index.html

```diff
@@ -0,0 +1,93 @@
+<div
+    tuiCard
+    tuiSurface
+    class="surface"
+    (click)="currentIndex = 0"
+>
+    <tui-svg
+        src="tuiIconGooglePay"
+        tuiSlot="top"
```

Comment:
Make sure you remove all the now unnecessary tuiSlot attributes.

Reply:
fixed